### DISCUSSION TITLE
New offline / online indicator for colorblind people (online indicator color/shape/size customizable)

### DISCUSSION THREAD

**Discussion by KrispyFunky** [rustdesk/rustdesk#5556] [General]
If I can make one small suggestion for us colorblind people.  The online status indicator that is either a very small little green dot or a very small little red dot is impossible to tell the difference for people who are red green colorblind... even harder since it is so small.

Are you able to make a different visual change to maybe the widows / operating system icon too that will show that it is offline.  Here is a mock up suggestion I have.  The one on the left is online and the one on the right is offline.  Appreciate your thoughts... since I have around 70 computers setup in my address book, this would be very helpful to quickly scan to see if I have a computer offline.

Here is my mock up idea.

https://imgur.com/a/rf8NJkN

Thanks

**Comment by mblyamaha** [comment]
It seems that there are too few colorblind users. I'm also struggling with this little green dot problem. It would be greatly appreciated if the color or the shape could be customized.

**Comment by My1** [comment]
maybe like anydesk where the indicator is larger and is either a red dot with nothng or a green one with a checkmark

**Comment by StratISaurus** [comment]
I'm only lightly red green colour blind, but it's hard to tell between the two shades. In my own platforms, I prefer grey / black to signify offline and a bright green for online. No ambiguity then. Ideally, a client side setting to customize that would be great.

**Comment by Chaoscontrol** [comment]
Agree, one more colourblind here. The dot is too small, and the 2 shades of green and red are not the best ones for contrast.

**Comment by Affection4036** [comment]
I have the same problem, it's impossible for me to see if a device is either online or offline.

**Comment by ruuster13** [comment]
Just adding my voice to the choir of red-green colorblind people who are unable to make sense of the status indicator lights.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/9544
